### ISSUE TITLE
Tagbar like open fold

### ISSUE THREAD

**Issue by amikai** [liuchengxu/vista.vim#153]
A lot of icon is good (g:vista#renderer#icons). 
But i like tagbar just use ▼ to open, ▶ to fold. It's clearly and simple.
And the foldtext behavior like tagbar is more sensible (the foldtext vista used is rough now).

How about add g:tagbar_like_open_fold variable to support that.

**Comment by liuchengxu** [comment]
Can you convince me by some pictures and more details? I personally rarely notice these things.

**Comment by amikai** [comment]
![image](https://user-images.githubusercontent.com/13815956/65574776-e61e0800-dfa0-11e9-9d73-ed4191b77a5b.png)
Just like tagbar. when open, the icon is ▶. when closed, the icon is  ▼.
And the foldtext like the tagbar in picture.

**Comment by liuchengxu** [comment]
I'll take a look when I get some free time.

**Comment by amikai** [comment]
Can you put the #84 test file on gist  ? I want to test the behavior in tagbar.

**Comment by liuchengxu** [comment]
No available test file :(. It's merely a POC and has been put on hold. The scope kind means `functions`, `variables` in the above pic.

**Comment by amikai** [comment]
After #84, I don't know why conflict. Maybe i haven't got the point.
Can you explain detail for that?

**Comment by liuchengxu** [comment]
The problem is that the scope kind may have too many icons, that's a bit of distracting.

![image](https://user-images.githubusercontent.com/8850248/65579169-cf2fe380-dfa9-11e9-82ac-483ecbab7177.png)

**Comment by amikai** [comment]
Yes, it is very distracting.
I think most people just want to read the roughly structure of whole program in tree structure(like #84).
So, maybe create one variable `g:vista_open_fold_icon_only`. when this variable is set, show ▼ and ▶ only, and hide other icon.

**Comment by liuchengxu** [comment]
#163 has refined the fold display. But I can't guarantee it works exactly same with tagbar as I personally do not rely on the outline view recently.
